### PR TITLE
Fix bugs in first run of bastion.

### PR DESCRIPTION
**Pull request by jamielennox**
Fixup bugs in running bastion for the first time. 

The /etc/ansible/ directory doesn't exist to write an inventory too and the shade package can't be installed without some additional ubuntu packages.

### CODE REVIEW

File: provision.yml [hunk 1]
```diff
@@ -2,15 +2,14 @@
   hosts: localhost
   tasks:
     - os_server:
-        name: contrasjc-bastion
+        name: contrasjc-bastion-2
         flavor: m1.medium
-        image: 7a8169b8-96d4-48ab-a607-34f79dea4156 # Ubuntu 14.04
-        key_name: default
+        image: ubuntu-16.04-py27
+        key_name: jamielennox
```
Comment (j2sol) [inline provision.yml]: lol

**Comment by jamielennox** [comment]
-1 this for now, I was testing on 16.04 but I wasn't supposed to add provision or the systemD change to the p-r

**Comment by j2sol** [comment]
Looks like this is good to review again? The 16.0whatever stuff has been removed. @jamielennox is this good to go?

**Comment by jamielennox** [comment]
yep, this is good. I'll pass to adam to merge.

**Pull request merged**